### PR TITLE
[PBW-3370] Subscribe to the earlier STREAM_PAUSED event for mobile lo…

### PR DESCRIPTION
**Pull request by chibzu**
…ck screen scenarios.

### CODE REVIEW

**Comment by mistermendez** [comment]
LGTM

File: js/controller.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.subscribe(OO.EVENTS.CONTENT_TREE_FETCHED, 'customerUi', _.bind(this.onContentTreeFetched, this));
       this.mb.subscribe(OO.EVENTS.AUTHORIZATION_FETCHED, 'customerUi', _.bind(this.onAuthorizationFetched, this));
       this.mb.subscribe(OO.EVENTS.PLAYING, 'customerUi', _.bind(this.onPlaying, this));
-      this.mb.subscribe(OO.EVENTS.PAUSED, 'customerUi', _.bind(this.onPaused, this));
+      this.mb.subscribe(OO.EVENTS.STREAM_PAUSED, 'customerUi', _.bind(this.onPaused, this));
```
Comment (aldigun) [inline js/controller.js]: Should not used STREAM_PAUSED. This is not a public event and is already removed with the next VTC feature.

Comment (chibzu) [inline js/controller.js]: Well, that makes this fix more interesting then.

Comment (aldigun) [inline js/controller.js]: We can stay as is using STREAM_PAUSED, but will need to change later on.

**Pull request merged**